phpBB/phpbb/report/controller/report.php

```diff
@@ -0,0 +1,268 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\report\controller;
+
+use phpbb\exception\http_exception;
+use phpbb\exception\runtime_exception;
+
+class report
+{
+	/**
+	 * @var \phpbb\config\db
+	 */
+	protected $config;
+
+	/**
+	 * @var \phpbb\user
+	 */
+	protected $user;
+
+	/**
+	 * @var \phpbb\template\template
+	 */
+	protected $template;
+
+	/**
+	 * @var \phpbb\controller\helper
+	 */
+	protected $helper;
+
+	/**
+	 * @var \phpbb\request\request_interface
+	 */
+	protected $request;
+
+	/**
+	 * @var \phpbb\captcha\factory
+	 */
+	protected $captcha_factory;
+
+	/**
+	 * @var string
+	 */
+	protected $phpbb_root_path;
+
+	/**
+	 * @var string
+	 */
+	protected $php_ext;
+
+	/**
+	 * @var array
+	 */
+	protected $tmp_vars;
```

Comment:
remove it and pass it to the protected method as an argument. Btw use a better name please